RELEASE.md

```diff
@@ -1,3 +1,28 @@
+# Release 1.5.1
+
+NOTICE: TensorBoard 1.6.0+ will move to the `tensorboard` package name on PyPI,
+instead of using `tensorflow-tensorboard`. Only bugfix updates on 1.5.x will be
+applied to the old package name. To upgrade to TensorBoard 1.6.0+ please *first*
+`pip uninstall tensorflow-tensorboard` before doing `pip install tensorboard`.
+
+The 1.5 minor series tracks TensorFlow 1.5.
+
+## Bug fixes
+
+- #554 - line charts no longer "shrink" after run changes on other tabs
+- #889 - fix xComponentsCreationMethod default in vz-line-chart
+- #898 - fixed offset of checkbox label in projector dashboard - thanks @marcj
```

Comment:
nit: Maybe use present tense for all bullet points.

Reply:
This is just a holdover from the original PR/commit message tense, but fair enough.  Will update.

Reply:
Fixed to use past tense consistently.